### DISCUSSION TITLE
Dropdown and translations

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10976] [Archive]
Main table: `project`
with a dropdown: project's `stage` (sorted according to a stage's `sort` field)

I'd like the stages to appear in the user's language.

Should I create a `stage` **Collection** with `label`, `sort` columns and a `translations` relationship?
Then, how can I show only the user's language stages in the dropdown?

**Comment by rijkvanzanten** [comment]
There's currently no way to translate the values of a dropdown interface

**Reply by MarcHagen** [reply]
Will that be a feature in the future ?
~~Or is there a way to still get, in this case `stage`, translated via a hidden collection ?~~

What i've ended up with is a seperate data model called `employment_type` with tranlations below it.
Then Many to One relation in my `work_history` model. Still have to fiqure out how to get the correct translations but looks promising :)

**Reply by rijkvanzanten** [reply]
Yeah! We have some concepts and R&D in the works to make any interface/field option translatable 🙂
